setup.cfg

```diff
@@ -67,6 +67,16 @@ docs =
   sphinx-gallery
   sunpy-sphinx-theme
   towncrier
+  reproject
+  astroquery
+  jplephem
+format =
```

Comment:
Are we using this for anything?

Reply:
No I just thought that pydoc style would use this in the future to check th entire code base using all these packages.

Reply:
Is it possible to make pre-commit do all this?

Reply:
In the future I don't see why not.

Reply:
I meant we can make pre-commit install these things, so do we really need a new extra for them?

Reply:
Oh I thought I removed them.

Reply:
Removed.